clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/util/UpsertIdGenerator.scala

```diff
@@ -50,7 +50,7 @@ object UpsertIdGenerator {
   /**
     * Return a random index into `Encoding`.
     */
-  private val next7bitIndex = (_: Int) => source.nextInt(Encoding.length)
+  private val next6bitIndex = (_: Int) => source.nextInt(Encoding.length)
```

Comment:
Would it be hard to pick a name that describes what this is without using the value that the name holds?

Reply:
It's really a dumb function introduced to work around lack of a single-argument iterate functional, which could take a lambda. Suggest a name if there is one you like better.